features/unit/abijson.feature

```diff
@@ -86,3 +86,30 @@ Feature: AbiJson
     Examples:
       | methodsig                | name            | description                 | network1                                     | network1-app-id | network2                                     | network2-app-id | jsonfile      | directory         |
       | add(uint32,uint32)uint32 | ExampleContract | This is an example contract | wGHE2Pwdvd7S12BL5FaOP20EGYesN73ktiC1qzkkit8= | 1234            | SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI= | 5678            | contract.json | abi_responsejsons |
+
+
+  Scenario Outline: Get a method by name from Interface
+    When I create the Method object from method signature "<methodsig>"
+    And I create another Method object from method signature "<extramethod>"
+    And I create an Interface object from the Method object
+    And I get the method from the Interface by name "<name>"
+    Then the produced method signature should equal "<methodsig>". If there is an error it begins with "<error>"
+
+    Examples:
+      | methodsig                | extramethod              | name | error           |
+      | add(uint32,uint32)uint32 |                          | add  |                 |
+      | add(uint32,uint32)uint32 |                          | sub  | found 0 methods |
+      | add(uint32,uint32)uint32 | add(uint64,uint64)uint64 | add  | found 2 methods |
+
+  Scenario Outline: Get a method by name from Contract
+    When I create the Method object from method signature "<methodsig>"
+    And I create another Method object from method signature "<extramethod>"
```

Comment:
```suggestion
    When I append to my Method objects list in the case of a non-empty signature "<methodsig>"
    And I append to my Method objects list in the case of a non-empty signature "<methodsig>"
```

nit: we can consolidate the add/create method step definitions into a single one. See also my upcoming comment on the adjoining py-sdk PR.

Reply:
I need it to be a distinct column so I can test same name but different selector

Reply:
Right, I meant to make the second one take `<extramethod>` ... fixed